addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -459,6 +459,42 @@ def _rtc_invite_members(self, member_ids=None):
                     ),
                 },
             )
+            devices, private_key, public_key = self.channel_id._get_web_push_parameters(members.partner_id.ids)
+            if devices:
+                if self.channel_id.channel_type != 'chat':
+                    icon = "/web/image/discuss.channel/%d/avatar_128" % self.channel_id.id
+                elif guest := self.env["mail.guest"]._get_guest_from_context():
+                    icon = "/web/image/mail.guest/%d/avatar_128" % guest.id
+                elif partner := self.env.user.partner_id:
+                    icon = "/web/image/res.partner/%d/avatar_128" % partner.id
+                self.channel_id._push_web_notification({
+                    "title": _("Incoming call"),
+                    "options": {
+                        "body": _("Conference: %(channel_name)s", channel_name=self.channel_id.display_name),
+                        "icon": icon,
+                        "vibrate": [100, 50, 100],
+                        "requireInteraction": True,
+                        "tag": f"call_{self.channel_id.id}",
```

Comment:
We might want to have a function returning this value to avoid potential mismatch when cancelling over code refactoring.